File: java/javadpbo/src/com/sem4dpbo/week1/praktikum/latihanpraktikum1b/DataTim.java
package com.sem4dpbo.week1.praktikum.latihanpraktikum1b;

public class DataTim {
    //atribut untuk menampung data
    private String nama, negara, kota, kandang, tahunBerdiri, pemain;
    private int nomorPunggung;
    //constructor
    public DataTim(
            String nama, String negara, String kota, String kandang, String tahunBerdiri, String pemain,
            int nomorPunggung)
    {
        this.nama = nama;
        this.negara = negara;
        this.kota = kota;
        this.kandang = kandang;
        this.tahunBerdiri = tahunBerdiri;
        this.pemain = pemain;
        this.nomorPunggung = nomorPunggung;
    }

    //method setter
    public void setNama(String nama) { this.nama = nama;}

    public void setNegara(String negara) {this.negara = negara;}

    public void setKota(String kota) {this.kota = kota;}

    public void setKandang(String kandang) {this.kandang = kandang;}

    public void setTahunBerdiri(String tahunBerdiri) {this.tahunBerdiri = tahunBerdiri;}

    public void setPemain(String pemain) { this.pemain = pemain; }

    public void setNomorPunggung(int nomorPunggung) {this.nomorPunggung = nomorPunggung;}

    //method getter
    public String getNama() { return nama; }

    public String getNegara() { return negara; }

    public String getKota() { return kota; }

    public String getKandang() { return kandang; }

    public String getTahunBerdiri() { return tahunBerdiri;}

    public String getPemain() { return pemain; }

    public int getNomorPunggung() {return nomorPunggung;}

}

File: cpp/1st_Week/Praktikum/Latihan1b/DataTim.cpp
#include <iostream>
#include <string>

using namespace std;

class DataTim
{
    //private properties
    private:
        //variabel menampung data tim
        string nama, negara, kota, kandang, tahunBerdiri, pemain;
        int nomorPunggung;
    public:
        //constructor kosong dan yang memakai paramter
        DataTim(){};

        DataTim(
            string paramNama, string paramNegara, 
            string paramKota, string paramKandang, 
            string paramTahunBerdiri, string paramPemain,
            int paramNomorPunggung)
        {
            nama = paramNama;
            negara = paramNegara;
            kota = paramKota;
            kandang = paramKandang;
            tahunBerdiri = paramTahunBerdiri;
            pemain = paramPemain;
            nomorPunggung = paramNomorPunggung;
        }

        //method setter
        void setNama(string paramNama) { nama = paramNama;}

        void setNegara(string paramNegara) {negara = paramNegara;}

        void setKota(string paramKota) { kota = paramKota;}

        void setKandang(string paramKandang) {kandang = paramKandang;}

        void setTahunBerdiri(string paramTahunBerdiri) {tahunBerdiri = paramTahunBerdiri;}

        void setPemain(string paramPemain) { pemain = paramPemain; }

        void setNomorPunggung(int paramNomorPunggung) { nomorPunggung = paramNomorPunggung;}


        //method get
        string getNama() { return nama; }

        string getNegara() { return negara; }

        string getKota() { return kota; }

        string getKandang() { return kandang; }

        string getTahunBerdiri() { return tahunBerdiri;}

        string getPemain() { return pemain; }

        int getNomorPunggung() {return nomorPunggung;}

        //destructor
        ~DataTim(){};
};